shared/src/test/scala/singleton/ops/ImplicitConversionSpec.scala

```diff
@@ -11,5 +11,5 @@ class ImplicitConversionSpec extends Properties("ImplicitConversion") {
   property("Wrong OpToOp") = {illTyped("""val ret : 2 + 1 = implicitly[4 - 2]"""); true}
   property("Wrong OpToSingleton") = {illTyped("""val ret : 3 = implicitly[4 - 2]"""); true}
   property("Wrong SingletonToOp") = {illTyped("""val ret : 2 + 1 = 4"""); true}
-  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueOf[4 - 1]}
-}
\ No newline at end of file
+  property("ValueOf[Op]") = wellTyped {val ret : 3 = valueof.valueOf[4 - 1]}
```

Comment:
Let me suggest another thing. You can pull #33 ' s branch to a new branch locally, and try to apply the changed that make scala-js work. If everything is OK, then let me know and I will merge #33. If you encounter difficulties, then we apply your PR first, and I will handle the conflicts later.